### PR TITLE
Create a run script and update the readme file

### PR DESCRIPTION
**Pull request by anjula-sack**
- Create a run script and update the readme file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "start": "http-server --port 9000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/anjula-sack/sack-site.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/anjula-sack/sack-site/issues"
```
Comment (jayasanka-sack) [inline package.json]: Replace `anjula-sack` with `acf-sack`

Check other places too

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "sack-site",
+  "version": "1.0.0",
+  "description": "Official Website of St. Anthony's College Kandy",
+  "main": "jQuery.scrollSpeed.js",
```
Comment (jayasanka-sack) [inline package.json]: Remove this line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "http-server --port 9000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/anjula-sack/sack-site.git"
+  },
+  "author": "",
```
Comment (jayasanka-sack) [inline package.json]: Antonian Computer Fraternity - St. Anthony's College, Kandy

File: package.json (hunk 1 above)
Comment (anjula-sack) [reply package.json]: done

**Pull request closed without merging**